nemo/core/optim/lr_scheduler.py

```diff
@@ -322,6 +324,31 @@ def _get_lr(self, step):
         return new_lrs
 
 
+# TODO: Can we add something like this to be consistent with Optimizer?
```

Comment:
Yes it was planned, I can do it now since team is fine with such a deffered instantiation pattern